mgmt_sdk_compiler.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+for i in `ls -d */*/v20* | grep -v "node_modules/*/*"`; 
+do 
```

Comment:
Why does this need to be dynamic? Is there not a reason you cannot create a pom file and commit it?

Reply:
new SDK versions are added every time a team creates a new version in the rest API... if we have a static pom file, well need to update that as well upon any new API version additions.This way the build dynamically picks up all SDK versions.